swagger.yaml

```diff
@@ -1,22 +1,25 @@
 swagger: '2.0'
 info:
   title: Analytics Core API
-  version: '0.1'
+  version: '0.2'
 basePath: /api/v1/
+host: "f8a-analytics-2445582058137.production.gw.apicast.io"
 consumes:
   - application/json
 produces:
   - application/json
+
+securityDefinitions:
+  tokenAuth:
+    type: apiKey
+    name: user_key
+    in: query
+    description: Token Auth is used for Authentication
```

Comment:
It would be good if the `description`  explicitly mentions it deals with 3scale token.

Reply:
Ack!